SaintCoinach/Definitions/SpearfishingItem.json

```diff
@@ -37,6 +37,10 @@
         "target": "TerritoryType"
       }
     },
+    {
+      "index": 6,
+      "name": "folklore"
```

Comment:
This doesn't follow naming convention and it looks like it's wrong, this column is always 0 so we can't assume what it's supposed to be.